crates/fluvio-controlplane-metadata/src/connector/spec.rs

```diff
@@ -30,6 +30,74 @@ pub struct ManagedConnectorSpec {
 
     pub secrets: BTreeMap<String, SecretString>,
 }
+#[derive(Debug, PartialEq, Clone)]
+#[cfg_attr(
+    feature = "use_serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase", untagged)
+)]
+pub enum ConnectorVersionInner {
+    String(String),
+    Option(Option<String>),
+}
+
+impl Default for ConnectorVersionInner {
+    fn default() -> Self {
+        Self::String(String::new())
+    }
+}
+impl From<String> for ConnectorVersionInner {
+    fn from(inner: String) -> Self {
+        Self::String(inner)
+    }
+}
+impl ToString for ConnectorVersionInner {
+    fn to_string(&self) -> String {
+        match self {
+            ConnectorVersionInner::String(inner) => inner.to_string(),
+            ConnectorVersionInner::Option(inner) => inner.clone().unwrap_or_else(|| "".to_string()),
```

Comment:
this is for the old connectors right? shouldn't it be `latest` by default?

Not sure what is the expected response to `fluvio connector config old-connector` when it was created with the previos version.